docs/man/man1/openshift-infra-f5-router.1

```diff
@@ -68,6 +68,10 @@ You may restrict the set of routes exposed to a single project (with \-\-namespa
     List of comma separated domains to deny in routes
 
 .PP
+\fB\-\-disable\-namespace\-ownership\-check\fP=false
+    Disables the namespace ownership checks for a route host with different paths or for overlapping host names in the case of wildcard routes
```

Comment:
This is generated from the `pkg/cmd/infra/router/f5.go` and the prior one from `pkg/cmd/admin/router/router.go` - will make sure the text syncs up.